scipy/stats/_multivariate.py

```diff
@@ -1717,6 +1719,12 @@ class wishart_gen(multi_rv_generic):
     distribution :math:`W_1(\nu, 1)` collapses to the :math:`\chi^2(\nu)`
     distribution.
 
+    The algorithm [2]_ implemented by the `rvs` method occasionally
+    produces numerically singular matrices with :math:`p - 1 < \nu < p`; the
```

Comment:
```suggestion
    The algorithm [2]_ implemented by the `rvs` method may
    produce numerically singular matrices with :math:`p - 1 < \nu < p`; the
```
Based on [this information](https://github.com/scipy/scipy/pull/13393#issuecomment-763176173), how does this wording look @mckib2?

Reply:
Looks good to me!